.github/workflows/ispc-ci-san.yml

```diff
@@ -0,0 +1,43 @@
+name: Sanitizers
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+env:
+  LLVM_REPO: https://github.com/ispc/ispc.dependencies
+
+jobs:
+  linux-build-ispc-llvm15-asan:
+    runs-on: ubuntu-latest
+    env:
+      LLVM_VERSION: "15.0"
+      LLVM_TAR: llvm-15.0.7-ubuntu18.04-Release+Asserts-x86.arm.wasm.tar.xz
+      ISPC_OPAQUE_PTR_MODE: "ON"
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+
+    - name: Install dependencies
+      run: |
+        .github/workflows/scripts/install-build-deps.sh
+
+    # We will use system gcc until we rebuild LLVM with compiler-rt.
+    - name: Check environment
+      run: |
+        ./check_env.py
+        which -a gcc
+        cat /proc/cpuinfo
+
+    - name: Build package
```

Comment:
`Build package` is misleading name here, as only config is done. I suggest keeping the name, but doing package build here as well.